hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/Options.java

```diff
@@ -518,4 +522,94 @@ public String toString() {
     MD5MD5CRC,  // MD5 of block checksums, which are MD5 over chunk CRCs
     COMPOSITE_CRC  // Block/chunk-independent composite CRC
   }
+
+  /**
+   * The standard {@code openFile()} options.
+   */
+  @InterfaceAudience.Public
+  @InterfaceStability.Evolving
+  public static final class OpenFileOptions {
```

Comment:
It may be time to break this file out into multiple files, each with one class.

Reply:
did have that initially, then pulled back in. No real preference